src/averageAges.js

```diff
@@ -15,11 +15,25 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  return people
+    .filter(({ sex, died }) => {
+      const deathCentury = Math.ceil(died / 100);
+      const isCenturyRequired = century
+        ? sex === 'm' && deathCentury === century
+        : sex === 'm';
```

Comment:
It would be cool to put all the "magic values" in constants. This applies to all code.

Reply:
done